schema/spacewalk/common/views/rhnChannelNewestPackageView.sql

```diff
@@ -52,9 +52,11 @@ FROM (
                 FROM rhnPackageEVR pe
                     INNER JOIN susePackageExcludingPartOfPtf p ON p.evr_id = pe.id
                     INNER JOIN suseChannelPackageRetractedStatusView cp ON cp.package_id = p.id
-                    LEFT JOIN suseAppstreamPackage appstreampkg ON appstreampkg.package_id = p.id
-                    LEFT JOIN suseAppstream appstream ON appstream.channel_id = cp.channel_id
-                        AND appstream.id = appstreampkg.module_id
+                    LEFT JOIN (
+                        select a.id as id, a.channel_id as channel_id, ap.package_id as package_id
+                        from suseAppStreamPackage ap
+                        inner join suseAppStream a on a.id = ap.module_id
```

Comment:
Can you please keep the style consistent with the surrounding code? I know we have no SQL style guide that tells us to use upper or lower case for built-ins, but it would be good to stick to one, at least for any given file.